docs/extensions/cross-chain-data/example_pt_2.mdx

```diff
@@ -0,0 +1,366 @@
+---
```

Comment:
Should everything in this document that says "oracle" change to "event listener"?

In the extensions overview, we define an oracle as "event listener + resolution". In this page, we describe the event listeners. It seems weird to call these oracles here?